tests/acceptance/bootstrap/RevaContext.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use TestHelpers\AppConfigHelper;
+use TestHelpers\SetupHelper;
+
+require_once 'bootstrap.php';
+
+/**
+ * Context for Reva specific steps
+ */
+class RevaContext implements Context {
+
+    /**
+     * @var FeatureContext
+     */
+    private $featureContext;
+
+    /**
+     * @BeforeScenario
+     *
+     * @param BeforeScenarioScope $scope
+     *
+     * @return void
+     * @throws Exception
+     */
+    public function setUpScenario(BeforeScenarioScope $scope) {
+        // Get the environment
+        $environment = $scope->getEnvironment();
+        // Get all the contexts you need in this context
+        $this->featureContext = $environment->getContext('FeatureContext');
+        SetupHelper::init(
```

Comment:
Codacy found an issue: [Avoid using static access to class '\TestHelpers\SetupHelper' in method 'setUpScenario'.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5940619)